gapic/src/main/com/google/gapid/views/SettingsDialog.java

```diff
@@ -137,6 +141,9 @@ protected void configureDialog(FileDialog dialog) {
       allowUpdateChecks = withLayoutData(
           createCheckbox(this, Messages.UPDATE_CHECK_OPTION, true),
           withSpans(new GridData(SWT.LEFT, SWT.TOP, false, false), 2, 1));
+      allowUpdateCheckPrereleases = withLayoutData(
+          createCheckbox(this, Messages.UPDATE_CHECK_PRERELEASE_OPTION, false),
```

Comment:
Indent this checkbox, so it's clear that it's a dependent option.

Reply:
done